### PR TITLE
create tx component refactored, new RpcApi.call2() method

### PR DESCRIPTION
**Pull request by gagarin55**
- tx/create.js file refactored into CreateTxForm (presentation) and CreateTx (container)
- RpcApi.call() returns Promise which reject in case json rpc response contains 'error' field.

### CODE REVIEW

**Comment by gagarin55** [comment]
@elaineo Could you please review this PR

File: src/lib/rpcApi.js [hunk 1]
```diff
@@ -11,6 +11,29 @@ export class RpcApi {
...
+  }
+
+  /**
+   * This call analyses JSON RPC response.
+   * It returns promise which resolves whether 'result' field found
+   * or reject in case 'error' field found.
+   *
+   * @returns {Promise}
+   */
+  call2(name, params, headers) {
```
Comment (elaineo) [inline src/lib/rpcApi.js]: maybe a more descriptive name than call2?

Comment (gagarin55) [inline src/lib/rpcApi.js]: I totally agree, the question is which version of call should we use? 
- old one which just returns json and UI layer has to analyze json
- call2 which try to understand json response and reject promise if error found

Comment (elaineo) [inline src/lib/rpcApi.js]: I prefer `call2`. I guess we'll have a later PR to replace all the `call` with `call2`

Comment (splix) [inline src/lib/rpcApi.js]: I prefer reject if error found

Comment (gagarin55) [inline src/lib/rpcApi.js]: Don't merge PR, please, I will improve it

**Comment by gagarin55** [comment]
I've replaced all calls to new RpcApi.call()

**Pull request merged**